doc/ExpertOptions.md

```diff
@@ -689,7 +689,7 @@ CWL provides a different set of runtime attributes used in the [ResourceRequirem
 ### Additional DNAnexus-specific runtime settings
 
 There are several parameters that also affect the runtime behavior of an applet and can be supplied upon its creation in its [dxapp.json file](https://documentation.dnanexus.com/developer/apps/app-metadata#annotated-example):
-* `runSpec.executionPolicy`: Specifies when to try to automatically restart failed jobs, and how many times
+* `runSpec.executionPolicy`: Specifies when to try to automatically restart failed jobs, and how many times.  Note that applet's restartOn setting will be overridden by the parent job's executionPolicy setting when launched from a job during workflow execution.
```

Comment:
Alternative:
If a job is launched form a running (parent) job, then applet's `restartOn` setting will not take any effect and will be overridden by the parent job's `executionPolicy`.